services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -51,14 +53,28 @@ const Asset = ({ libraryData }) => {
   const [assetData] = libraryData.assets
   const { name, description, inherits: inheritsData, thumbnailData: imageData } = assetData.content
 
+  const id = libraryData.content.id
+
+  const libraryId = id
```

Comment:
Have you found `library.content.id` to not be slugified? If that's the case, see `getSlug()` in slug.js utils. Pass in content and it'll slugify id, with name as a fallback.